docs/content/latest/explore/ysql-language-features/queries.md

```diff
@@ -840,14 +846,16 @@ The `EXISTS` operator has the following syntax:
 EXISTS (SELECT 1 FROM some_table WHERE condition);
 ```
 
-The following example applies the `EXISTS` operator to two tables:  `employees` and  `salary`. The example shows how to use the `EXISTS` operator on the `employee_no` column in a manner similar to an inner join. This query can return no more than one row for each row in the `employees` table, even though there are corresponding rows in the `salary` table.
+The following example applies the `EXISTS` operator to two tables (the  `employees` table and a hypothetical  `salary` table) and shows how to use `EXISTS` on the `employee_no` column in a manner similar to an inner join:
 
 ```sql
 SELECT name FROM employees
   WHERE EXISTS (SELECT 1 FROM salary
                   WHERE salary.employee_no = employees.employee_no);
 ```
 
+If the `salary` table existed, the preceding query would have returned no more than one row for each row in the `employees` table, even though there would have been corresponding rows in the `salary` table.
+
 ## Recursive Queries and CTEs
 
 Common Table Expressions (CTEs) allow you to execute recursive, hierarchical, and other type of complex queries in a simplified manner by breaking down these queries into smaller units. A CTE exists only during the query execution and represents a temporary result set that you can reference from another SQL statement.
```

Comment:
```suggestion
Common Table Expressions (CTEs) allow you to simplify recursive, hierarchical, and other types of complex queries by breaking them down into smaller units. A CTE exists only during query execution, and is a temporary result set that you can reference from another SQL statement.
```

Reply:
Reworded this sentence on purpose (at the expense of elegance)